docs/kustomize_to_docs.sh

```diff
@@ -1,13 +1,11 @@
 #!/usr/bin/env bash
 set -ex pipefail
 
-source $LOCALBIN/.virtualenv/bin/activate
-
 BAREMETAL=docs/assemblies/ref_example-OpenStackDataPlaneNodeSet-CR-for-bare-metal-nodes.adoc
 FOOTER=$(sed '0,/----/d' $BAREMETAL | sed -e '0,/----/d')
 sed -i '/----/q' $BAREMETAL
 sed -i 's/preprovisioned/baremetal/' examples/no_vars_from/kustomization.yaml
-$KUSTOMIZE --load-restrictor LoadRestrictionsNone build examples/no_vars_from | yq ' select(.kind == "OpenStackDataPlaneNodeSet")' -Y >> $BAREMETAL
```

Comment:
the -Y by the way is what makes this incompatioable

the python version now required -y or -Y  when modifying a yaml input to specify that
it should render yaml output and if you want to preserve the original order ectra during the transformation

the go version does not support those flags and does not require them

Reply:
Yeah, that is true.